Replaced .0 with .x for doc version drop down

PaarthAgarwal opened the pull request:
Signed-off-by: PaarthAgarwal <paarth.agarwal.mec20@itbhu.ac.in>
Updates the config.toml file to display the version number with .x instead of .0 in the drop-down.
Partially fixes #219.

Vad1mo commented:
I would suggest using it without any suffix, like. **2.4** ...**2.8**.

PaarthAgarwal commented on config.toml:
```diff
@@ -21,47 +21,47 @@ youtube_video_id = "4zZiBcvZmgQ"
 
 [[params.versions]]
-harborversion = "2.7.0"
+harborversion = "2.7.x"
```
Done

OrlinVasilev commented:
I'm not sure if that's ok due to the fact we do versioning on the docs in branches, it breaks the links now 
[Details](https://deploy-preview-414--goharbor-io.netlify.app/)
check the preview first

OrlinVasilev commented:
![image](https://user-images.githubusercontent.com/7236111/218098070-17f20083-7e59-4ec1-a14f-64caa86ab168.png)

OrlinVasilev commented:
https://deploy-preview-414--goharbor-io.netlify.app/ here is the address

PaarthAgarwal commented:
> I'm not sure if that's ok due to the fact we do versioning on the docs in branches, it breaks the links now [Details](https://deploy-preview-414--goharbor-io.netlify.app/) check the preview first

That's true. It breaks the links indeed

PaarthAgarwal commented:
@OrlinVasilev @Vad1mo 
I've updated the PR. Instead of modifying config.toml I've modified navbar.html and menu.html to display versions without the last ".0". Have a look
After the fix:
![Screenshot from 2023-02-14 04-44-04](https://user-images.githubusercontent.com/86092410/218595976-98d10802-7d0b-4384-b14c-99958afc309e.png)

Vad1mo commented:
@OrlinVasilev can you take a look at it again to see if it works for you?

OrlinVasilev commented:
it works but it redirects to 2.7.0 again which is what we expect anyway!
Let me test a bit more ... thanks!
